core/src/main/scala/org/genivi/sota/core/PackagesResource.scala

```diff
@@ -112,7 +77,7 @@ class PackagesResource(resolver: ExternalResolverClient, db : Database)
         completeOrRecoverWith(
           for {
             _ <- resolver.putPackage(ns, pid, description, vendor)
-            (uri, size, digest) <- savePackage(pid, fileData)
+            (uri, size, digest) <- packageStorage.store(pid, fileData)
```

Comment:
yeah `packageStore.store` will save it either to s3 or local fs, depending on config.